packages/chatpal-search/client/style.css

```diff
@@ -0,0 +1,304 @@
+.chatpal-admin-link {
+	color:red !important;
+	text-decoration: underline !important;
+}
+
+.chatpalProvider .search-form {
+	padding: 0 24px;
+}
+
+.chatpal_search-loader {
+	position: relative;
+	margin-top: 50px;
+}
+
+.chatpal-search-container {
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+}
+
+.chatpal-search-result {
+	position: relative;
+	overflow: auto;
+	overflow-x: hidden;
+	padding: 0 24px 24px;
+	background-color: #f1f1f1;
+	flex: 1;
+}
+
+.chatpal-search-typefilter li {
+	cursor: pointer;
+	display: inline-block;
+	width: 49%;/*WHY? Fix*/
```

Comment:
can you use flexbox here? `display: flex` on the parent and `flex: 0 0 50%` here